tests/arch/arm/arm_interrupt/README.txt

```diff
@@ -1,11 +1,18 @@
-Title: Test to verify code fault handling in ISR execution context (ARM Only)
+Title: Test to verify code fault handling in ISR execution context
+       and the behavior of irq_lock() and irq_unlock() when invoked
+       from User Mode. (ARM Only)
 
 Description:
 
-This test verifies that we can handle system fault conditions
+The first test verifies that we can handle system fault conditions
 while running in handler mode (i.e. in an ISR). Only for ARM
 Cortex-M targets.
 
+The second test verifies that threads in user mode are able to call
+irq_lock() and irq_unlock() functions without causing a CPU fault,
+however they will be able to properly read or modify the IRQ locking
```

Comment:
why do we need to prove that calling irq_lock() on arm *doesn't* generate a fault? I don't understand why we need this test

Reply:
Ups, correct, thanks. I'l rephrase this. What we need to prove is that the locking from user mode does not work _even though_ i does not strictly fail.